lib/consensus/raft/consensus.go

```diff
@@ -151,7 +151,7 @@ func NewConsensus(host host.Host, cfg *ClusterRaftConfig, mpool *messagepool.Mes
 	consensus.SetActor(actor)
 
 	peers := []peer.ID{}
-	addrInfos, err := addrutil.ParseAddresses(ctx, cfg.InitPeerset)
+	addrInfos, _ := addrutil.ParseAddresses(ctx, cfg.InitPeerset)
```

Comment:
We should check the error here too

Reply:
Same here — what should I do with it?

Reply:
`	if err != nil {
		return nil, err
	}`